### ISSUE TITLE
multiple branches

### ISSUE THREAD

**Issue by jasonebox** [wehrad/SICE#30]
is it confusing for users to have multiple branches?

**Comment by wehrad** [comment]
I don't think so. We have a master branch that is always ready for production, and other branches for various development strategies. This is a standard project organisation on Github.

**Comment by jasonebox** [comment]
how do users know multiple branches exist? How do they know which branch contains all the useful Readme content. I think you're making additional Readme content that is not in the main branch. That implies you need to merge your Readme with the master.

**Comment by wehrad** [comment]
Everything useful, ready to use, finished and optimized should be in the master branch, codes and documentation. But it looks like here you are talking about the [main repository](https://github.com/GEUS-SICE/SICE) more than the master branch of this repository?
